FirebaseDatabase/Tests/Integration/FIRDatabaseQueryTests.m

```diff
@@ -4419,7 +4419,7 @@ - (void)testGetUpdatesPersistenceCacheWhenEnabled {
   }
 }
 
-- (void)testGetSkipsPersistenceCacheWhenOnline {
+- (void)SKIPtestGetSkipsPersistenceCacheWhenOnline {
```

Comment:
I like the above comment. Does it also apply here?

```suggestion
// TODO: On arm hardware Macs, the following test hangs with the emulator, but passes with a real
// project.
- (void)SKIPtestGetSkipsPersistenceCacheWhenOnline {
```

Reply:
Thanks. This test has been flaking in nightlies as well.